docs-site/docs/user/analysis/working-with-workflow-executions.md

```diff
@@ -1,15 +1,64 @@
 ---
-sidebar_position: 3
+sidebar_position: 2
 id: working-with-workflow-executions
 title: Working with Workflow Executions
 ---
 
-<!-- TODO: Include the following headers with explanations and/or descriptions:
-- States of Workflow Executions
-- User Workflow Executions
-  - Navigating to the user workflow executions page
-  - Creating a user workflow execution
-- Automated Workflow Executions
-  - Navigating to the automated workflow executions page
-  - Creating an automated workflow execution
--->
+Learn how to use workflow executions in IRIDA Next
+
+## View User Workflow Executions
+
+There are two ways to navigate and view user workflow executions:
+
+  * If you are currently viewing a project or group, click the dropdown on the left sidebar that contains the project or group name, and click **Workflow Executions** in the dropdown menu.
+  * If you are not currently viewing a project or group, click **Workflow Executions** on the left sidebar.
+
+This page will list all your workflow executions.
+
+## View Automated Workflow Executions
+
+Prerequisites:
+  * You must have at least the Analyst role to the project with workflow executions.
+
+To view automated workflow executions:
+  1. Navigate to the [project](../project/projects/manage-projects#view-projects-you-have-access-to) containing the workflow executions
+  2. Click **Workflow Executions**
+
+This page will list all of the project's workflow executions.
+
+## View Single Workflow Execution
+
+To view a specific workflow execution:
+  1. Follow the steps to view either the [user](../analysis/working-with-workflow-executions#view-user-workflow-executions) or [automated](../analysis/working-with-workflow-executions#view-automated-workflow-executions) workflow executions listing page
+  2. Click the **ID** of the workflow execution you'd like to view
+
+Each individual workflow execution will contain a summary, the parameters selected during set-up, and the output files once analysis has completed.
```

Comment:
Josh's PR to add a samplesheet view to the workflow execution will be merged in before this, so we you update this to include `samplesheet including input files` or something similar

Reply:
Fixed in d26ef5bed67f85a18f9ac5c210e6869ba9966f5a. Let me know if the wording doesn't work for you